docs/source/release_notes/release-notes.rst

```diff
@@ -76,10 +76,10 @@ develop
          - :ref:`Sweep metrics <dropwizard-metrics>` now record counts of cell-timestamp pairs examined rather than the count of entire cells examined. This provides more accurate insight on the work done by the sweeper.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1911>`__)
 
-    *    - |userbreak|
-         - The Sweep CLI configuration parameters ``--batch-size`` and ``--cell-batch-size`` have been removed, as we now batch on cell-timestamp pairs rather than by rows and cells.
+    *    - |deprecated|
```

Comment:
We should collapse the two release notes. Not necessarily in this PR (it can happen after tagging)